CHANGELOG.md

```diff
@@ -16,3 +16,8 @@
 * [UIPFAUTH-21](https://issues.folio.org/browse/UIPFAUTH-21) The error "Something went wrong" appears when search by same search option and updated query
 * [UIPFAUTH-26](https://issues.folio.org/browse/UIPFAUTH-26) The pane size of the "Search & filter" pane changes when user changes scale of the screen
 * [UIPFAUTH-28](https://issues.folio.org/browse/UIPFAUTH-28) Browse: Remove the link action on a placeholder row
+
+## [0.1.0](https://github.com/folio-org/ui-plugin-find-authority/tree/v0.1.0) (2022-10-04)
```

Comment:
Maybe it should be `1.0.0` as in package.json?

Reply:
`1.0.0` is saved for a first release, but this is a pre-release for Technical Evaluation process so we use `0.1.0`